certificate-authentication.md

```diff
@@ -373,20 +361,18 @@ The output:
 
 ```
 --------------
-mysql  Ver 15.1 Distrib 10.4.10-MariaDB, for Linux (x86_64) using readline 5.1
+mysql  Ver 8.3.0 for Linux on x86_64 (MySQL Community Server - GPL)
```

Comment:
```suggestion
mysql  Ver 8.2.0 for Linux on x86_64 (MySQL Community Server - GPL)
```

TiDB v7.6.0 prints out 8.2.0 instead of 8.3.0. Please have a check.

```
tiup playground v7.6.0
mysql> \s
--------------
mysql  Ver 8.2.0 for macos14.0 on arm64 (Homebrew)
```

Reply:
On another machine it reports this:
```
mysql  Ver 8.1.0 for FreeBSD14.0 on amd64 (Source distribution)
```

This is the same line as what `mysql --version` reports.